nullaway/src/main/java/com/uber/nullaway/NullabilityUtil.java

```diff
@@ -213,7 +213,10 @@ public static Stream<? extends AnnotationMirror> getAllAnnotations(Symbol symbol
         elementValues.entrySet()) {
       ExecutableElement elem = entry.getKey();
       if (elem.getSimpleName().contentEquals("value")) {
-        return (String) entry.getValue().getValue();
+        Object value = entry.getValue().getValue();
+        if (value instanceof String) {
+          return (String) value;
```

Comment:
With this change, what happens to an `@Contract` annotation that doesn't have a `value` field or has a non-String `value` field?

Reply:
I believe it will be silently ignored; see response to next comment.